### PR TITLE
Improve reports page visual design

### PR DESCRIPTION
**Pull request by violetyao**
-  Add tables for vulnerabilities, deployment validations, post deploy jobs, merges to master, production promotions
-  Add links to cluster/namespace definitions
-  Add links to Grafana dashboards
-  For reports page, sort reports by date, then by name
-  For service page, sort reports by date
-  For service page, add "details" button to expand more reports
-  Add openshift console links to report page

### CODE REVIEW

File: src/pages/elements/Report.js [hunk 1]
```diff
@@ -22,39 +39,369 @@ const Vulnerabilities = ({vs}) => {
...
+              formatters: [headerFormat]
+            },
+            cell: {
+              formatters: [ns=>(<GrafanaContainerVulnerabilities namespace={ns} label="Grafana Dashboard" />) , cellFormat]
+            },
+            property: 'ns'
+          },
+          {
+            header: {
+              label: 'Console',
```
Comment (jmelis) [inline src/pages/elements/Report.js:330]: We don't need the `Console` column in the deployment validation table.

**Review by jmelis — CHANGES_REQUESTED** [review]
@violetyao this is amazing work!!

The code looks perfect! I just have a couple of comments about the content of the pages:

- Perhaps the latest report in the service page can be just a flat link, and not a table? We could even put it at the top, under Service Owners, with the same formatting. We can keep the "More reports" at the bottom.
- Could you also remove the Valet section from the report? That is unused and we are deprecating it.
- Let's remove the `console` column from the deploymentvalidator table

it looks great!

**Comment by violetyao** [comment]
Thank you Jaime!! I've implemented the following updates: 1) redesign reports section on services page 2) remove valet 3) remove console link in deployment validation table.

**Review by jmelis — APPROVED** [review]
LGTM! this is perfect, thanks a lot @violetyao

**Comment by violetyao** [comment]
Thank you Jaime!!

**Pull request merged**